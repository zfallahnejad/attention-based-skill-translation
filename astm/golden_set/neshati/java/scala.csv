247533,scala
298389,scala
9204,scala
334519,scala
515054,scala
1296806,scala
605744,scala
1632462,scala
788207,scala
2292812,scala
