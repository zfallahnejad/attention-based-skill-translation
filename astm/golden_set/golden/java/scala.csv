247533,64,47,0.5765765765765766
298389,26,14,0.65
9204,25,28,0.4716981132075472
334519,19,7,0.7307692307692307
515054,17,13,0.5666666666666667
1296806,16,16,0.5
605744,14,3,0.8235294117647058
1632462,13,5,0.7222222222222222
788207,12,10,0.5454545454545454
2292812,11,7,0.6111111111111112
220447,10,14,0.4166666666666667
406435,10,2,0.8333333333333334
1251870,10,5,0.6666666666666666
